Added age filter with range 1-20, 20-30, 30-40, 40-50, 50-60, 60-70 and 70 up

error-yatish opened the pull request:
This is a reference to [#112](https://github.com/someshkar/covid19india-cluster/issues/112)

sibeshkar commented:
Thanks. Few changes : 

1. Make the Age node Icons bigger and increase edge length
2. Add icon legend
2. Modify age filter icon - maybe a clock icon will do

error-yatish commented:
@sibeshkar I have committed below changes
1. Added Age node Icons and increased edge length
2. Added icons to legend
3. Changed age filter icon to Clock Icon

sampathBlam commented:
@error-yatish : 
Great implementation. Just a couple of suggestions. 
1.  All age groups seem to settle towards the centre of the cluster. 

![image](https://user-images.githubusercontent.com/17728976/77822484-2eb5aa00-7119-11ea-9634-40e162ec9c4e.png)

Can we move each group a bit away from each other so that each group is visible clearly? Like it happens for other filters? For eg, in case of state.

![image](https://user-images.githubusercontent.com/17728976/77822546-daf79080-7119-11ea-8011-b73ea0451409.png)

2. Can we group all cases whose age is not known, into a separate group ?

@sibeshkar : Your suggestions here would be appreciated as well.
